camel-sender-demo/pom.xml

```diff
@@ -14,7 +14,7 @@
 
     <properties>
 
-        <camel-quarkus.version>2.8.0</camel-quarkus.version>
+        <camel-quarkus.version>2.9.0</camel-quarkus.version>
         <camel-version>3.16.0</camel-version>
```

Comment:
The Camel version didn't change since `2.8.0`: https://github.com/apache/camel-quarkus/blob/2e300a4e0d6a9229593bb8e7479ad3866785a622/pom.xml#L42-L43